server/chat/backend/agent/tools/notion/postmortem.py

```
@@ -0,0 +1,606 @@
+) -> int:
+    """Create one Notion page per unchecked action item in the target DB.
+
+    Returns the number of pages successfully created.
+    """
+    items = parse_action_items(postmortem_md or "")
+    if not items:
+        return 0
+
+    try:
+        db_schema = client.get_database(action_items_database_id)
+    except Exception as exc:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)